src/runner.rs

```diff
@@ -177,16 +177,14 @@ fn name_end_char(c: char) -> bool {
     c == ' ' || c == '='
 }
 
-/// Check if an argument string affects the Firefox profile
+/// Check if an argument string matches a given name
 ///
 /// Returns a boolean indicating whether a given string
-/// contains one of the `-P`, `-Profile` or `-ProfileManager`
-/// arguments, respecting the various platform-specific conventions.
-pub fn is_profile_arg(arg: &str) -> bool {
+/// contains an argument of a given names, respecting
+/// various platform-specific conventions.
```

Comment:
That isn't really right. It assumes that you have an array of arguments and are passing each arg from the array down into the function. It won't tokenize an argument string into an array of arguments.

I also think that `matches` is a weird name, because it sounds like the result of a regex.